domains/certificates/Query.API/API/MeasurementsSyncer/SlidingWindowService.cs

```diff
@@ -1,19 +1,24 @@
+using System;
 using System.Collections.Generic;
 using System.Linq;
+using API.Configurations;
 using API.MeasurementsSyncer.Metrics;
 using DataContext.Models;
 using DataContext.ValueObjects;
 using Measurements.V1;
+using Microsoft.Extensions.Options;
 
 namespace API.MeasurementsSyncer;
 
 public class SlidingWindowService
```

Comment:
What about the filter method? I think it lets all measurements after the synchronization point through, but if I understand this PR, then missing intervals may exists after the synchronization point, and therefore also measurements that have been published already?

Reply:
The goal was to limit the dataset to be worked with by:

1. The Filter
2. The sliding window update method
3. The FindMeasurements method
4. The Find Missing Intervals method

If we make sure we limit the dataset provided to those 4 methods, by subtracting the time span ranging from the minimumAgeThreshold until currentTimestamp, from the "DateTo", then those measurements will be invisible,for the 4 methods.

Reply:
I think i just convinced myself that the filter method is working because of the restriction we assume in fetch, that no measurements after sync point is returned. Should we add this to the filter method? I mean add the age requirement as a parameter and filter out all measurements after the age threshold?

Reply:
That is already being done implicitly. We pass a modified DateFrom and DateTo variable, when attempting to fetch and filter measurements.

Reply:
We have added the logic to the filter method now:

https://github.com/Energinet-DataHub/energy-origin/pull/1575/commits/cae2c104906e72965e05ad3ef51ad1cf41abb641